app/sidekiq/decision_review/saved_claim_sc_status_updater_job.rb

```diff
@@ -85,6 +84,24 @@ def get_evidence_uploads_statuses(submitted_appeal_uuid)
       result
     end
 
+    def get_and_update_secondary_form_statuses(submitted_appeal_uuid)
+      all_complete = true
+      return all_complete unless Flipper.enabled?(:decision_review_track_4142_submissions)
+
+      secondary_forms = AppealSubmission.find_by(submitted_appeal_uuid:)&.secondary_appeal_forms
+      secondary_forms = secondary_forms&.filter { |form| form.delete_date.nil? } || []
```

Comment:
Not a huge fan of variable reuse here. Can we use a different variable or add the `.filter {}` to the end of the chain on the line above?

Reply:
I want to turn this into a scope on AppealSubmission so the whole query can happen in SQL, which should remove the need for this mess